system-variables.md

```diff
@@ -1861,7 +1861,7 @@ mysql> desc select count(distinct a) from test.t;
 - Persists to cluster: Yes
 - Type: Boolean
 - Default value: `ON`
-- Specifies whether to allow using outer table as the build side in outer join for TiFlash.
+- When the variable value is `ON`, the left join operator always uses inner table as the build side and the right join operator always uses outer table as the build side. If you set the value to `OFF`, outer join operations can use either side of the tables as the build side.
```

Comment:
operations -> operator

Reply:
If the value is set to `OFF`, outer join operator can use either side of the tables as the build side.

Reply:
```suggestion
- When the variable value is `ON`, the left join operator always uses inner table as the build side and the right join operator always uses outer table as the build side. If you set the value to `OFF`, the outer join operator can use either side of the tables as the build side.
```